internal/driver/driver.go

```diff
@@ -176,6 +178,12 @@ func applyCommandOverrides(cmd []string, v variables) variables {
 			v.set("nodecount", "80")
 		}
 	}
+
+	if outputFormat == report.Proto {
```

Comment:
Side question: why does "peek" case resets the filter but not not the tagfilter?

Reply:
It does seem odd. My best guess is that tagfilter was added later, and this is an oversight, but I'll look into the commit history to see if there's evidence for that.

Reply:
So we should fix it?

Reply:
Tag filtering was added after the peek command, (in [this](https://github.com/google/pprof/pull/33/files) PR), which suggests to me this is an oversight.

I've changed it in this PR. Let me know if you think it ought to be a separate PR.